Assets/Scripts/Models/Furniture.cs

```diff
@@ -20,7 +20,7 @@
 /// InstalledObjects are things like walls, doors, and furniture (e.g. a sofa).
 /// </summary>
 [MoonSharpUserData]
-public class Furniture : IXmlSerializable, ISelectable, IContextActionProvider
+public partial class Furniture : IXmlSerializable, ISelectable, IContextActionProvider
```

Comment:
although its worth noting that by having `partial` people will learn what the keyword means and how splitting up classes can be useful in a large project.